packages/mermaid/src/docs/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "docs",
+  "private": true,
+  "type": "module",
+  "scripts": {
+    "dev": "vitepress --port 3333 --open",
+    "build": "pnpm prefetch && vitepress build",
+    "build-no-prefetch": "vitepress build",
+    "serve": "vitepress serve",
+    "preview-https": "pnpm build && serve .vitepress/dist",
+    "preview-https-no-prefetch": "pnpm build-no-prefetch && serve .vitepress/dist",
+    "prefetch": "pnpm fetch-contributors && pnpm fetch-avatars",
+    "fetch-avatars": "esno .vitepress/scripts/fetch-avatars.ts",
+    "fetch-contributors": "esno .vitepress/scripts/fetch-contributors.ts"
+  },
+  "dependencies": {
+    "@vueuse/core": "^10.1.0",
+    "jiti": "^1.18.2",
+    "vue": "^3.2.47"
+  },
+  "devDependencies": {
+    "@iconify-json/carbon": "^1.1.16",
+    "@types/fs-extra": "^11.0.1",
+    "@unocss/reset": "^0.51.8",
+    "@vite-pwa/vitepress": "^0.0.5",
+    "@vitejs/plugin-vue": "^4.2.1",
+    "esno": "^0.16.3",
```

Comment:
Why this over `ts-node-esm`, which we're using in the root `package.json`?

Reply:
Initially copied over the code from vitest, but esno is infact faster than ts-node-esm. 
We can replace ts-node-esm with esno completely in a different PR.

Reply:
Sounds good!

Btw, it looks like `esno` now just redirects to `tsx`, so we should probably use that directly instead. See https://github.com/esbuild-kit/esno/commit/01065aa6eec022616550298e7781be81bb78575c

Reply:
Replacing ts-node-esm didn't work, as some weird `__name` is not defined error was thrown during mermaid runtime (the scripts ran fine, error was only when running `dev`). So I've just used ts-node-esm everywhere.